### ISSUE TITLE
Event types in event webhook table do not match event types in Events section

### ISSUE THREAD

**Issue by Jericho** [sendgrid/docs#5894]
I was under the impression that there were 5 delivery events and 6 engagement events and this seems to be corroborated by the fact that there are 11 types of events documented in the [events objects](https://sendgrid.com/docs/for-developers/tracking-events/event/#event-objects) table.

However I just noticed that there are 6 [delivery events](https://sendgrid.com/docs/for-developers/tracking-events/event/#delivery-events) and 6 [engagement events](https://sendgrid.com/docs/for-developers/tracking-events/event/#engagement-events) listed in the __Events__ section for a total of 12 types of events,

I'm confused. Which one is right?

**Comment by tparkersendgrid** [comment]
Hello,

Checking the delivery event differences between the documented objects in the event objects table compared to the events section, note that the "bounce" event is broken into two different sub events (and these are both documented on the events section, accounting for both "bounce" and "block").  A "bounce" will always be posted via an event notification webhook (as an object) as "event=bounce" but there is a sub bounce type of "type=bounced" or "type=blocked" which accounts for both of the "bounce" events.

In other words:

Bounce = "event=bounce, type=bounced" (permanent failure)
Block = "event=bounce, type=blocked" (temporary failure)

I can see how this may be confusing.  If you have any other questions, please reach out to the SendGrid product support team via support.sendgrid.com.

Thanks,
